{"uuid":"f7d53ca9-316d-492e-87a1-0156cbb72211","historyId":"97d429ce43fe3db569b5fe66c81eaeb5","fullName":"login.LoginTests.LoginCorrecto_Admin","labels":[{"name":"package","value":"login.LoginTests"},{"name":"testClass","value":"login.LoginTests"},{"name":"testMethod","value":"LoginCorrecto_Admin"},{"name":"parentSuite","value":"Suite Login"},{"name":"suite","value":"Tests de Login"},{"name":"subSuite","value":"login.LoginTests"},{"name":"host","value":"DESKTOP-MNG9D3P"},{"name":"thread","value":"3020@DESKTOP-MNG9D3P.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Inicio de sesión exitoso"},{"name":"epic","value":"Login"},{"name":"feature","value":"Validaciones del formulario de inicio de sesión"},{"name":"severity","value":"critical"}],"links":[],"name":"LoginCorrecto_Admin","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"No se redirigió al Dashboard correctamente expected [true] but found [false]","trace":"java.lang.AssertionError: No se redirigió al Dashboard correctamente expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat funciones.LoginActions.verificarDashboardVisible(LoginActions.java:47)\r\n\tat login.LoginTests.LoginCorrecto_Admin(LoginTests.java:48)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\r\n\tat org.testng.TestRunner.run(TestRunner.java:619)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Verifica que el usuario admin pueda iniciar sesión con credenciales válidas","steps":[],"attachments":[{"name":"\uD83D\uDCF8 Screenshot al fallar","source":"d929a3ff-baac-41fe-bc2a-6da43e77f128-attachment"}],"parameters":[],"start":1744649220541,"stop":1744649221090}